pyhf/readxml.py

```diff
@@ -11,6 +11,7 @@ def import_root_histogram(rootdir, filename, path, name):
     #import pdb; pdb.set_trace()
     #assert path == ''
     # strip leading slashes as uproot doesn't use "/" for top-level
+    if path is None: path = ''
```

Comment:
This was needed to handle situations where `HistoPath` wasn't included -- and in these cases, it's equivalent to `''`. This code does need to be fixed up more to normalize the XMLs better...

Reply:
maybe 

```
path = path or ''
```

is more pythonic?